bin/tldr

```diff
@@ -22,11 +22,15 @@ program
   .option('-e, --random-example', 'Show a random example')
   .option('-f, --render [file]', 'Render a specific markdown [file]')
   .option('-m, --markdown', 'Output in markdown format')
-  .option('-o, --os [type]', 'Override the operating system [linux, osx, sunos, windows]')
-  .option('--linux', 'Override the operating system with Linux')
-  .option('--osx', 'Override the operating system with OSX')
-  .option('--sunos', 'Override the operating system with SunOS')
-  .option('--windows', 'Override the operating system with Windows')
+  .option('-p, --platform [type]', `Override the current platform [${platforms.supportedPlatforms.join(', ')}]`);
+
+for (const platform of platforms.supportedPlatforms) {
+  program.option(`--${platform}`, `Override the platform with ${platform}`);
+}
```

Comment:
The list of options no longer matches the README / completion scripts, where now will see `darwin`, `macos`, and `win32` here, but are not completable.

Also, having them in any order looks weird, and would prefer to see them alphabetized here and also in the description for `--platform` so that it's easier to look for a specific platform:
```
  -p, --platform [type]    Override the current platform [osx, darwin, macos, linux, openbsd, sunos, android, windows,
                           win32]
  --osx                    Override the platform with osx
  --darwin                 Override the platform with darwin
  --macos                  Override the platform with macos
  --linux                  Override the platform with linux
  --openbsd                Override the platform with openbsd
  --sunos                  Override the platform with sunos
  --android                Override the platform with android
  --windows                Override the platform with windows
  --win32                  Override the platform with win32
```

Reply:
If possible, can you fix this alone? Directly, in this branch. Thanks in advance.